playwright.config.ts

```diff
@@ -36,6 +36,7 @@ export default defineConfig({
 		stderr: 'pipe',
 		env: {
 			PORT,
+			NODE_ENV: 'test',
```

Comment:
when running on CI, we `cp` the `.env.example` to `.env` and so we've got values staring with `MOCK_` which turns   on the [`shouldMock` ](https://github.com/epicweb-dev/epic-stack/blob/main/app/utils/providers/github.server.ts#L22) flag.

But when running e2e tests locally and having `.env` with real values for GitHub OAuth app, we need something else to indicate we want mock actions. Hence adding this env var